pom.xml

```diff
@@ -34,10 +34,28 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jakarta.json.version>2.0.0-RC1</jakarta.json.version>
+        <jakarta.json.version>2.0.0-RC2</jakarta.json.version>
         <jakarta.json.bind.version>2.0.0-RC1</jakarta.json.bind.version>
+        <jakarta.enterprise.cdi-api.version>3.0.0-M2</jakarta.enterprise.cdi-api.version>
```

Comment:
Can you please undo all of the CDI related changes? We already have that covered in PR #408

Reply:
Hi @aguibert,

May you please merge the PR #408 first?

Reply:
Yep, PR #408 is merged now so you can rebase this one